UnrealPlugin/README.md

```diff
@@ -10,337 +11,38 @@ It was tested with Unreal Engine 4.26.2, but should work with other engines as w
 
-Then open the .uproject file.
+## Project creation
 
-In the top left of the Unreal editor menu, click on File -> Package -> Target Configuration -> [game name]Server and then go to File -> Package -> Windowsx64. 
-You can now use this packaged version of your game server to [test with LocalMultiplayerAgent](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/locally-debugging-game-servers-and-integration-with-playfab) 
-or to use it directly with PlayFab by [creating a build](https://developer.playfab.com/) and then using [PlayFab MpsAllocatorSample](https://github.com/PlayFab/MpsSamples/blob/master/MpsAllocatorSample/README.md).
+Unreal GSDK is installed into an Unreal server project. You will need a network-enabled multiplayer Unreal project with a dedicated-server mode. If you do not have one, you can follow our [Unreal prerequsitie setup](ThirdPersonMPSetup.md) guide. Once you have a network-enabled, multiplayer game, with a dedicated server, return here and continue.
 
-## Setting up a Windows Dedicated Server on PlayFab
-An important note about this is that you need to set the start command in process-based mode to \<root folder\>\\Binaries\\Win64\\\<project name\>Server.exe (or however the executable is called in the folder).
+## Project GSDK setup
 
-In container-based mode use \<mount folder\>\\\<root folder\>\\Binaries\\Win64\\\<project name\>Server.exe (or however the executable is called in the folder).
+With a multiplayer project configured for both a network connected game-client and game-server, you're ready to [install the GSDK](ThirdPersonMPGSDKSetup.md). This will allow your server to run on PlayFab Multiplayer Services.
 
-**If you use the executable in the root folder, the server will fail to initialize with the PlayFab system.**
+## Unreal Project Build Configurations
 
-## Setting up a Linux Dedicated Server on PlayFab
-During testing the following Dockerfile + startup.sh script worked excellent with the PlayFab Linux VM:
+Once the GSDK is installed and configured in your project, you can [build your Unreal project](ThirdPersonMPBuild.md). There are multiple build configurations you will need to execute the full range of tests in future steps.
 
-### The Dockerfile:
-```Dockerfile
-FROM ubuntu:18.04
+## Test Local Server Deployment
 
-# Unreal refuses to run as root user, so we must create a user to run as
-# Docker uses root by default
-RUN useradd --system ue
-USER ue
+Once you have any client build, and a "Development Server" build, you can [test the MPS deployment process locally](ThirdPersonMPLocalDeploy.md), and verify your server works.
 
-EXPOSE 7777/udp
+## Deploy to PlayFab
```

Comment:
before deploying to PlayFab, it would be great if we insisted on using the project with LocalMultiplayerAgent. Testing locally saves time and money, since i) takes ~5-10 minutes to get a VM on PlayFab plus ii) this VM costs real money.

Reply:
"Test Local Server Deployment" is the name of the previous section, and covers exactly what you say.